examples/ai-core/src/generate-object/amazon-bedrock.ts

```diff
@@ -5,7 +5,9 @@ import { z } from 'zod';
 
 async function main() {
   const result = await generateObject({
-    model: bedrock('anthropic.claude-3-5-sonnet-20240620-v1:0'),
+    model: bedrock(
+      `arn:aws:bedrock:${process.env.AWS_REGION}:${process.env.AWS_ACCOUNT_ID}:inference-profile/us.anthropic.claude-3-5-sonnet-20240620-v1:0`,
```

Comment:
Can we use the simple ids here?

Reply:
Reverting this and others, although it seems it varies by account and model-enablement status. Perhaps there is an AWS UI I haven't found to let me enable simple ids for my account.